Source/Scene/Model.js

```diff
@@ -1933,6 +1931,11 @@ function parseTextures(model, context, supportsWebP) {
       supportsWebP
     ) {
       imageId = texture.extensions.EXT_texture_webp.source;
+    } else if (
+      defined(texture.extensions) &&
+      defined(texture.extensions.KHR_texture_basisu)
```

Comment:
Should `supportsBasis` be checked here as well?

Reply:
Added